.env.example

```diff
@@ -2,4 +2,4 @@ APP_PORT=5000
 APP_HOST=http://localhost
 DB_HOST=localhost
 DB_USER=dbuser
-DB_PASS=secret
\ No newline at end of file
+DB_PASS=your-password-here
```

Comment:
Add a single empty line at the end of this file. It's called a Line break, it would take out the "visible" stop character you can see here.